ui/ducks/send/send.js

```diff
@@ -2804,7 +2806,8 @@ export function signTransaction(history) {
 
     await dispatch(actions.setPrevSwapAndSend(prevSwapAndSendData));
 
-    if (stage === SEND_STAGES.EDIT) {
+    // you can only edit a basic send transaction
+    if (stage === SEND_STAGES.EDIT && !isSwapAndSend) {
```

Comment:
Does this mean we can remove the prev swap send state from redux as well?

Reply:
In theory but I’d rather keep it for now since we may be reutilizing it p soon. A bit of a boat anchor that we can clear pending future discussion on confirmations